### PR TITLE
add johnny to ggxrd-r2

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

**Comment by komidore64** [comment]
typo fix

File: src/data/guilty_gear_xrd_rev_2/johnny.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        - 'j.214[S]'
+    - name: Cancel
+      inputs: H
+      conditions: Mist Stance
+    - name: Mist Step
+    - inputs:
+      - 66
+      - 44
+      conditions: Ground Mist Stance
+    - name: High Mist Finer
```
Comment (yithian) [inline src/data/guilty_gear_xrd_rev_2/johnny.yml]: The high/mid/low Mist Finer attacks are technically on the release of the button held to enter mist stance.

Let's change the inputs for them to be `]P[`, `]K[` and `]S[`. And make them `conditions: Mist Stance`. Same for the air versions, with `conditions: Air Mist Finer stance`.

Comment (komidore64) [reply src/data/guilty_gear_xrd_rev_2/johnny.yml]: So the entire input of High Mist Finer (`214P`) is executed immediately upon release of whatever attack button is being held to maintain Mist Stance?

Comment (komidore64) [reply src/data/guilty_gear_xrd_rev_2/johnny.yml]:
```
08:16 <@achvatal> so for mist finer
08:16 <@achvatal> 214keydown puts him into mist stance
08:17 <@achvatal> while holding the button, he can shimmy around and whatnot
08:17 <@achvatal> then when he lets go of whatever button he pressed to enter the stance, he does mist finer attack
08:17 <adamprice> ohhhh, and then 'key-up' -> attack triggers
08:17 <adamprice> got it
08:18 <adamprice> okay that's weird
08:18 <@achvatal> but yeah, if you just do regular 214button, it's the ~same thing
08:18 <adamprice> :|
```

Comment (komidore64) [reply src/data/guilty_gear_xrd_rev_2/johnny.yml]: updated

**Pull request merged**